Vagrantfile

```diff
@@ -134,14 +154,48 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   #
   #   chef.validation_client_name = "ORGNAME-validator"
 
+  # Use masakari_ci_conf section to get masakari source from custom source.
+  # "masakari_git" : Repo to clone masakari source
+  #                  default is https://github.com/ntt-sic/masakari.git
+  # "masakari_branch" : Branch to checkout in above repo
+  #                     default is master
+  # Here is a sample for controller
+  # chef.json = {
+  #     "controller_conf" => {
+  #       "host_ip" => "192.168.50.10",
+  #       "service_host" => "192.168.50.10"
+  #     },
+  #     "masakari_controller_conf" => {
+  #       "hostentrylist" => [
+  #                           { "compute01" => "192.168.50.11" },
+  #                           { "compute02" => "192.168.50.12" }
+  #                          ]
+  #     },
+  #     "masakari_ci_conf" => {
+  #       "masakari_git" => "https://github.com/ntt-sic/masakari.git",
+  #       "masakari_branch" => "masater"
+  #     }
+  #   }
+
+
+
   # Definition of Controller node
-  config.vm.define "controller" do |controller|
-    controller.vm.hostname = "controller"
+  config.vm.define "controller01" do |controller|
+    controller.vm.hostname = "controller01"
```

Comment:
If you change the hostname, you should adjust the description in README.md as well.